loadtest/store/memstore/store.go

```diff
@@ -75,6 +75,31 @@ func (s *MemStore) SetChannel(channel *model.Channel) error {
 }
 
 func (s *MemStore) SetChannelMembers(channelId string, channelMembers *model.ChannelMembers) error {
-	s.channelMembers[channelId] = channelMembers
+	membersMap := make(map[string]*model.ChannelMember)
+	members := *channelMembers
```

Comment:
i know this is silly, but we have a pointer de-reference without checking for nil...

Reply:
> i know this is silly, but we have a pointer de-reference without checking for nil...

It's not silly at all but it's all over the store code at this time. I want to make a dedicate ticket to review everything.

Reply:
Done https://mattermost.atlassian.net/browse/MM-20958